### PR TITLE
Add BOASwap documents

### PR DESCRIPTION
**Pull request by TrustHenry**
For BOASwap

### CODE REVIEW

**Review by Kang-Simon — APPROVED** [review]

**Comment by TrustHenry** [comment]
I will modify the image path and rebuild it.

File: config/en/mkdocs.yml [hunk 1]
```diff
@@ -90,3 +90,16 @@ nav:
             - Guide: biznet/wallet/guide.md
             - Key Management: biznet/wallet/key-management.md
             - Metamask: biznet/wallet/tutorials/metamask.md
+    - BOASwap:
+        - Introduction: boa_swap/introduction.md
+        - Connecting to a Wallet: boa_swap/wallet/connectwallet.md
+        - Myassets:
```
Comment (MichaelKim20) [inline config/en/mkdocs.yml]:
```suggestion
        - My Assets:
```

File: config/kr/mkdocs.yml [hunk 2]
```diff
@@ -90,3 +90,14 @@ nav:
             - 가이드: biznet/wallet/guide.md
             - 키관리: biznet/wallet/key-management.md
             - 메타마스크: biznet/wallet/tutorials/metamask.md
+    - BOASwap:
+        - 소개: boa_swap/introduction.md
+        - 월렛 연결하기: boa_swap/wallet/connectwallet.md
+        - 나의자산: boa_swap/myassets/myassets.md
```
Comment (MichaelKim20) [inline config/kr/mkdocs.yml]:
```suggestion
        - 나의 자산: boa_swap/myassets/myassets.md
```

**Review by MichaelKim20 — CHANGES_REQUESTED** [review]
확인해 주세요

File: config/en/mkdocs.yml (hunk 1 above)
Comment (TrustHenry) [reply config/en/mkdocs.yml]: The suggestion applied.

File: config/kr/mkdocs.yml (hunk 2 above)
Comment (TrustHenry) [reply config/kr/mkdocs.yml]: The suggestion applied.

**Review by MichaelKim20 — APPROVED** [review]

**Pull request merged**